hackathon-website/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import Navbar from "./Components/Navbar/Navbar";
+import "./App.css";
+import SideDrawer from "./Components/SideDrawer/SideDrawer";
+import Backdrop from "./Components/Backdrop/Backdrop";
+import ImagineMare from "./Components/ImagineMare/ImagineMare";
+import Parteneri from "./Components/Parteneri/Parteneri";
+
+const App = () => {
+  const [sideDrawerOpen, setSideDrawerOpen] = useState(false);
+
+  const drawerToggleClickHandler = () => {
+    const negateSideDrawerOpen = !sideDrawerOpen;
+    setSideDrawerOpen(negateSideDrawerOpen);
```

Comment:
Redundant lines. Poti sa scrii direct:
` setSideDrawerOpen(!sideDrawerOpen);`

Reply:
Stiu ca domnu de la tutorial o zis ca la class components ii bine sa creezi o alta variabila la setState si m-am gandit ca si aici e nevoie

Reply:
La this.setState() da, asa se face. Dar tu aici ai in primul rand o componenta functionala, deci nu mai exista acea functie, si in al doilea rand tu te folosesti de hooks, de useState. E putin diferit contextul, si aici e ok si fara o variabila in plus din moment ce ai un setter dedicat variabilei sideDrawerOpen.